per_sdk/svm/dummy_tx.py

```diff
@@ -0,0 +1,175 @@
+import argparse
+import hashlib
+import logging
+import struct
+import urllib
+
+import httpx
+from solana.rpc.async_api import AsyncClient
+from solana.transaction import Transaction
+from solders.instruction import AccountMeta, Instruction
+from solders.message import Message
+from solders.pubkey import Pubkey
+from solders.system_program import ID as system_pid
+from solders.sysvar import INSTRUCTIONS as sysvar_ixs_pid
+
+from per_sdk.svm.helpers import configure_logger, read_kp_from_json
+
+logger = logging.getLogger(__name__)
+
+DEADLINE_MAX = 2**63 - 1
+
+
+async def main():
```

Comment:
function is a bit too long. you can extract the parser part into a separate function.